src/context/MyProvider.js

```diff
@@ -70,109 +71,125 @@ class MyProvider extends Component {
-						fileName: "New Document",
-						userId: this.userId,
-					}),
-				})
-					.then((data) => data.json())
-					.then((data) => {
-						data.created.time = this.ConvertDate(
-							data.created.timestamps.updatedAt
-						);
-						this.setState({
-							documents: [...this.state.documents, data.created],
-						});
-						resolve(data.created.fileId);
+		// if (this.guest !== "Yes") {
+		return new Promise((resolve, reject) => {
+			fetch(this.apiUrl + "preview/createFile/", {
+				method: "PATCH",
+				headers: {
+					Authorization: this.token,
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify({
+					fileName: "New Document",
+					userId: this.userId,
+				}),
+			})
+				.then((data) => data.json())
+				.then((data) => {
+					data.created.time = this.ConvertDate(
+						data.created.timestamps.updatedAt
+					);
+					this.setState({
+						documents: [...this.state.documents, data.created],
 					});
-			});
-		} else {
-			let newDoc = {
-				fileName: "NewDocument",
-				time: "Today",
-				fileData: "hello",
-				fileId: "newDoc",
-				_id: "newDoc",
-			};
-			this.setState({
-				documents: [...this.state.documents, newDoc],
-			});
-		}
+					resolve(data.created.fileId);
+				});
+		});
+		// } else {
+		// 	let newDoc = {
+		// 		fileName: "NewDocument",
+		// 		time: "Today",
+		// 		fileData: "hello",
+		// 		fileId: "newDoc",
+		// 		_id: "newDoc",
+		// 	};
+		// 	this.setState({
+		// 		documents: [...this.state.documents, newDoc],
+		// 	});
+		// }
 	};
 	TDocumentHandler = (content, title) => {
 		console.log("TEMPLATE", content, title);
-		if (this.guest !== "Yes") {
-			return new Promise((resolve, reject) => {
-				fetch(this.apiUrl + "preview/createFile/", {
-					method: "PATCH",
-					headers: {
-						Authorization: this.token,
-						"Content-Type": "application/json",
-					},
-					body: JSON.stringify({
-						fileName: title,
-						userId: this.userId,
-						fileData: content,
-					}),
-				})
-					.then((data) => data.json())
-					.then((data) => {
-						data.created.time = this.ConvertDate(
-							data.created.timestamps.updatedAt
-						);
-						console.log(data);
-						// let tdata = {
-						//   fileData: content,
-						//   fileId: data.created.fileId,
-						//   fileName : "Title",
-						//   time: data.created.time,
-						//   timestamps: data.created.timestamps
-						// }
-						this.setState({
-							documents: [...this.state.documents, data.created],
-						});
-						resolve(data.created.fileId);
-					});
-			});
-		} else {
-			return new Promise((resolve, reject) => {
-				let newDoc = {
+		// if (this.guest !== "Yes") {
+		return new Promise((resolve, reject) => {
+			fetch(this.apiUrl + "preview/createFile/", {
+				method: "PATCH",
+				headers: {
+					Authorization: this.token,
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify({
 					fileName: title,
-					time: "Today",
+					userId: this.userId,
```

Comment:
Here also

Reply:
Token is being used in the backend to get the userId, no point in sending this here

Reply:
Removed